ethereum/core/src/main/java/org/hyperledger/besu/ethereum/storage/keyvalue/WorldStateKeyValueStorage.java

```diff
@@ -78,6 +78,11 @@ private Optional<Bytes> getTrieNode(final Bytes32 nodeHash) {
     }
   }
 
+  @Override
+  public Optional<Bytes> getUnSafeTrieNode(final Bytes key) {
+    return keyValueStorage.get(key.toArrayUnsafe()).map(Bytes::wrap);
+  }
```

Comment:
a style suggestion, rename method to `getTrieNodeUnsafe` to uniform with other _unsafe_ methods like `toArrayUnsafe`

Reply:
done